easybuild/easyconfigs/h/HBase/HBase-1.0.2.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'PackedBinary'
+
+name = 'HBase'
+version = '1.0.2'
+
+homepage = 'http://hbase.apache.org/'
+description = """Apache HBase. is the Hadoop database, a distributed, scalable, big data store. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['%(namelower)s-%(version)s-bin.tar.gz']
```

Comment:
Yes that is a bash script. However I we would like to build it from source we would need to add an easyblock (like the hadoop easyblock) I assume and not use a toolchain, as it will use Maven tool to build the sources.